### PR TITLE
ezc3d: add version 1.3.8

### PR DESCRIPTION
**Pull request by qchateau**
Generated and committed by [Conan Center Bot](https://github.com/qchateau/conan-center-bot)
Find more updatable recipes in the [GitHub Pages](https://qchateau.github.io/conan-center-bot/)

Specify library name and version:  **ezc3d/1.3.8**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by theirix — CHANGES_REQUESTED** [review]
Need to export `EZC3D_SHARED` in package_info

**Review by SSE4 — APPROVED** [review]

**Review by SSE4 — CHANGES_REQUESTED** [review]
please add macro

**Review by jgsogo — APPROVED** [review]
This PR is adding a version, we can modify the `conanfile.py` in the next one.

**Review by uilianries — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Review by theirix — DISMISSED** [review]

**Review by intelligide — DISMISSED** [review]

**Comment by SpaceIm** [comment]
> 
> 
> Need to export `EZC3D_SHARED` in package_info

No, it's already defined in a header file: https://github.com/pyomeca/ezc3d/blob/7d40969ac2ee08ec4092504fc937d5f49e40e745/include/ezc3dConfig.h.in#L4

File: recipes/ezc3d/all/conandata.yml [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+  "1.3.8":
+    url: "https://github.com/pyomeca/ezc3d/archive/Release_1.3.8.tar.gz"
+    sha256: "888cb191c7939b31891e93356aa5bfcf21edb8785d3b5d4d9ec215984bc8b83e"
   "1.3.7":
     url: "https://github.com/pyomeca/ezc3d/archive/Release_1.3.7.tar.gz"
     sha256: "17cd3d4e20840710d58b8987c01ef0f880bc0c063ab52e0cb86ba41665ec33a0"
 patches:
+  "1.3.8":
+    - patch_file: "patches/fix-export-symbols.patch"
+      base_path: "source_subfolder"
```
Comment (SpaceIm) [inline recipes/ezc3d/all/conandata.yml:11]:
```suggestion
```

Comment (SpaceIm) [reply recipes/ezc3d/all/conandata.yml:11]: this patch was submitted to upstream and is included in 1.3.8:
https://github.com/pyomeca/ezc3d/blob/Release_1.3.8/CMakeLists.txt
https://github.com/pyomeca/ezc3d/blob/Release_1.3.8/include/ezc3dConfig.h.in

Comment (SpaceIm) [inline recipes/ezc3d/all/conandata.yml:11]:
```suggestion
```

this patch was submitted to upstream and is included in 1.3.8:
https://github.com/pyomeca/ezc3d/blob/Release_1.3.8/CMakeLists.txt
https://github.com/pyomeca/ezc3d/blob/Release_1.3.8/include/ezc3dConfig.h.in

See those warnings in logs:

```
[1m[33mWARN: patches/fix-export-symbols.patch: file 1/2:	 b'CMakeLists.txt'[0m
[1m[33mWARN: patches/fix-export-symbols.patch:  hunk no.1 doesn't match source file at line 43[0m
[1m[33mWARN: patches/fix-export-symbols.patch:   expected: b'    add_library(${PROJECT_NAME} SHARED ${SRC_LIST})'[0m
[1m[33mWARN: patches/fix-export-symbols.patch:   actual  : b'    set(EZC3D_SHARED TRUE)'[0m
[1m[33mWARN: patches/fix-export-symbols.patch: already patched  b'CMakeLists.txt'[0m
[1m[33mWARN: patches/fix-export-symbols.patch: file 2/2:	 b'include/ezc3dConfig.h.in'[0m
[1m[33mWARN: patches/fix-export-symbols.patch:  hunk no.1 doesn't match source file at line 4[0m
[1m[33mWARN: patches/fix-export-symbols.patch:   expected: b'// dllexport/import declaration'[0m
[1m[33mWARN: patches/fix-export-symbols.patch:   actual  : b'#cmakedefine EZC3D_SHARED'[0m
[1m[33mWARN: patches/fix-export-symbols.patch: already patched  b'include/ezc3dConfig.h.in'[0m
```

I recommend to always check diff between new version and last version in CCI.

**Review by SpaceIm — CHANGES_REQUESTED** [review]

**Review by prince-chrismc — APPROVED** [review]
@SpaceIm thank you for fixing upstream :heart:

https://github.com/pyomeca/ezc3d/commit/c7d98d71a088709d79e505b3726151eead2f3efd

**Review by SSE4 — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Pull request merged**